test/jdk/ProblemList.txt

```diff
@@ -789,6 +793,7 @@ java/awt/FullScreen/TranslucentWindow/TranslucentWindow.java 8258103 linux-all
 java/awt/Focus/FrameMinimizeTest/FrameMinimizeTest.java 8016266 linux-x64
 
 ############################################################################
-# java/foreign
+
+# jdk_foreign
```

Comment:
Please remove this hunk

Reply:
The heading on the sections are usually test group names so "jdk_foreign" is correct, but I don't know what this change is doing in this PR.

Reply:
Thanks guys for pointing this out to a newbie! Would you expect me to add a NOOP PR with correct labeling?
Regards
Thomas

Reply:
> Thanks guys for pointing this out to a newbie! Would you expect me to add a NOOP PR with correct labeling?
> Regards
> Thomas

You can if you want to. I'm unemotional :-)

If you do, you can mark it as "trivial" (as in, mention in the PR description its a "Trivial" change, which - when accepted by reviewers - means just one review is needed and it is exempt from the usual "wait 24hrs before pushing" rule.

Reply:
So if @AlanBateman does not insist, I'd keep it as is and hopefully rember for next time ;-)

Reply:
I don't have a string opinion either way. I think the main thing is that I suspect you'll need to exclude more tests as there are several tests for DatagramChannel that join multicast groups and some of these likely assume they can join an IPv4 multicast group when cases where the DatagramChannel is created with the no-arg open method (and IPv6 is enabled) or with the IPV6 protocol family.

Reply:
I've created https://bugs.openjdk.org/browse/JDK-8313404 to extract the fix for jdk_foreign from this PR. It's cleaner, also for backporting, although it incurs some process overhead.

Reply:
As discussed, please remove this part of the change.